### ISSUE TITLE
Data Center Picker: Update "Choose a data center instead" to use tertiary button style

### ISSUE THREAD

**Issue by danielbachhuber** [Automattic/wp-calypso#71140]
Currently looks like this:

<img width="752" alt="image" src="https://user-images.githubusercontent.com/36432/207388335-7b066f8a-b68d-4e50-9666-05ad0dba0842.png">

Suggested to look like this:

![image](https://user-images.githubusercontent.com/36432/207388574-3d936ab5-b54f-432d-8136-ae8f9863c5ec.png)

However, blocked by this: pdKhl6-1hu-p2#comment-1611

**Done is:**
* Button is updated to use the tertiary button style.

**Comment by danielbachhuber** [comment]
Punting on this change. It's very low priority in the grand scheme of things. We can pick it back up when it's relevant again.
